shell/app-shell/elements/test/fake-database-test.js

```diff
@@ -90,4 +90,4 @@ describe('FakeDatabase', function() {
       assert.equal(db.lastUpdate, values);
     });
   });
-});
+});
```

Comment:
We have end-with-newline bouncing back and forth between changes often (seen in a number of files in this PR for example), if it's easy to tweak your editor or whatever could be worth doing. It seems ending with newline is recommended, I found https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Not sure what our clang-format config specifies but I bet it has them.

Reply:
Ok, my bad. I usually try to remove all extra chars, but VSCode as `insertFinalNewline` option which I have set to true.